zenodo/modules/records/serializers/schemas/datacite.py

```diff
@@ -444,6 +446,41 @@ def inject_type(c):
 
         return items
 
+    def get_descriptions(self, obj):
+        """Get descriptions."""
+        items = []
+        desc = obj['metadata']['description']
+        max_descr_size = current_app.config.get(
+            'DATACITE_MAX_DESCRIPTION_SIZE', 20000)
+        if desc:
+            items.append({
+                'description': desc[:max_descr_size],
+                'descriptionType': 'Abstract'
+            })
+        notes = obj['metadata'].get('notes')
+        if notes:
+            items.append({
+                'description': notes[:max_descr_size],
+                'descriptionType': 'Other'
+            })
+        refs = obj['metadata'].get('references')
+        if refs:
+            items.append({
+                'description': json.dumps({
+                    'references': [
+                        r['raw_reference'] for r in refs
+                        if 'raw_reference' in r]
+                })[:max_descr_size],
+                'descriptionType': 'Other'
+            })
+        method = obj['metadata'].get('method')
+        if method:
+            items.append({
+                'description': method[:max_descr_size],
+                'descriptionType': 'method'
```

Comment:
I think this should be capitalized, same as the rest of the `descriptionType`:

```suggestion
                'descriptionType': 'Method'
```